src/rss/rss.ts

```diff
@@ -30,19 +30,19 @@ export default class RSS {
       const feed = await this.parser.parseURL(url);
 
       const converter = new TurndownService();
-      for (const item of feed.items) {
+      for (const item of feed.items ?? []) {
         const creator = item.creator || '';
         const link = item.link || '';
         let content = item.content || '';
-        const postDate = new Date(item.isoDate) || new Date();
+        const postDate = item.isoDate ? new Date(item.isoDate) : new Date();
```

Comment:
isn't this the same as `const postDate = new Date(item.isoDate);`

Reply:
Probably, but the strict typing won't allow it

Reply:
very weird